### ISSUE TITLE
Spacing out the buttons in source control [Accessibility]

### ISSUE THREAD

**Issue by haleyngonadi** [microsoft/vscode#140950]
Is it possible to better space out the buttons that appear under Source Control? Whenever I'm trying to mass commit files, I keep hitting the revert button, when I'm actually trying to hit the + button. 

![Screen Shot 2022-01-18 at 20 05 50](https://user-images.githubusercontent.com/6877114/150044433-e43ed6c2-1816-4592-8964-15401af140c6.png)

**Comment by isidorn** [comment]
@lszomoru no, we have not looked into it. And I do agree this is not an SCM problem but a general action bar problem.
This is the first feedback we have received about not enough space. I suggest to keep this open to gather more feedback.

@misolori for thoughts

**Comment by misolori** [comment]
Yea this affects all of our views with actions and I believe this is done intentionally to compactly fit the actions within the list item that has height of 22px while other areas (like the editor toolbar) are a few pixel wider and have some padding. I think the recent updates that show the hover state substantially improve this and we could try to make minor adjustments (like increase the padding from 2px to 3px).

**Comment by lszomoru** [comment]
@daviddossett assigning g this to you based on the comments above.
